src/settings/Validation/engine/handlers.js

```diff
@@ -14,6 +14,10 @@ export const isNotEmpty = (value) => {
   return isNumber(value) || !isEmpty(value);
 };
 
+export const isNotEmptyWithoutSpace = (value) => {
+  return typeof value === 'string' && value.trim();
```

Comment:
nit: => `&& !!value.trim()`

Reply:
I am not sure that it is necessary because we have implicit type conversion with `&&` operator

Reply:
fixed